hack/tests/e2e-upgrade.sh

```diff
@@ -11,6 +11,7 @@ cleanup() {
     stop_monitoring
     make artifacts
     make delete
+    az group delete -g "$RESOURCEGROUP" --yes --no-wait
```

Comment:
I really dont like now this no-wait as it makes next `/retest` broken. I would prefer for now blocking those but test results would not give failure with "delayed time to retest"